plugins/filter_log_to_metrics/log_to_metrics.c

```diff
@@ -1037,13 +1116,24 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct log_to_metrics_ctx, emitter_name),
      "Name of the emitter (advanced users)"
     },
-
     {
      FLB_CONFIG_MAP_SIZE, "emitter_mem_buf_limit", FLB_MEM_BUF_LIMIT_DEFAULT,
      0, FLB_TRUE, offsetof(struct log_to_metrics_ctx, emitter_mem_buf_limit),
      "set a buffer limit to restrict memory usage of metrics emitter"
     },
-
+    {
+      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
```

Comment:
I'd name these `flush_interval_sec` and `flush_interval_nsec` respectively and clarify that the second one is measured in nanoseconds.

TBH, what I'd really like to see is a new iteration of `FLB_CONFIG_MAP_TIME` that can handle smaller units and writes to a `struct flb_time` but I understand that it's wildly out of scope here so I'm only sharing it for context.